CONTRIBUTING.md

```diff
@@ -32,7 +32,11 @@ To run all unit tests:
 npm test
 ```
 
-The tests expect you to have Redis running locally on the default port, 6379. One way to do this is with Docker: `docker run -d -p 6379:6379 redis`
+The tests expect you to have Redis running locally on the default port, 6379. One way to do this is with Docker:
+
+```
+docker run -p 6379:6379 redis
+```
```

Comment:
This is both for better formatting and to remove the `-d` that isn't a necessary part of the command.